values.yaml

```
@@ -187,7 +187,7 @@ airflow:
 
   config:
     api:
-      auth_backend: astronomer.flask_appbuilder.current_user_backend
+      auth_backend: '{{- ternary "astronomer.flask_appbuilder.current_user_backend" "airflow.api.auth.backend.deny_all" .Values.useAstroSecurityManager }}'
```

nit, probably not required
```suggestion
      auth_backend: '{{ ternary "astronomer.flask_appbuilder.current_user_backend" "airflow.api.auth.backend.deny_all" .Values.useAstroSecurityManager }}'
```